<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EquivSolver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="FieldView" name="circuitGraphicsView">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>248</width>
          <height>667</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelection"/>
   <addaction name="actionDraw"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionI"/>
   <addaction name="actionE"/>
   <addaction name="actionR"/>
   <addaction name="actionC"/>
   <addaction name="actionL"/>
   <addaction name="actionGnd"/>
   <addaction name="separator"/>
   <addaction name="actionSolve"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElements">
    <property name="title">
     <string>Elements</string>
    </property>
    <addaction name="actionI"/>
    <addaction name="actionE"/>
    <addaction name="actionR"/>
    <addaction name="actionC"/>
    <addaction name="actionL"/>
    <addaction name="actionGnd"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelection"/>
    <addaction name="actionDraw"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuSolver">
    <property name="title">
     <string>Solver</string>
    </property>
    <addaction name="actionSolve"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionField_Size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuElements"/>
   <addaction name="menuSettings"/>
   <addaction name="menuSolver"/>
  </widget>
  <action name="actionI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/dc.png</normaloff>:/circuit_icons/resources/dc.png</iconset>
   </property>
   <property name="text">
    <string>I (Currency Source)</string>
   </property>
   <property name="toolTip">
    <string>I (Currency Source)</string>
   </property>
   <property name="shortcut">
    <string>Shift+I</string>
   </property>
  </action>
  <action name="actionE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/battery.png</normaloff>:/circuit_icons/resources/battery.png</iconset>
   </property>
   <property name="text">
    <string>E (Battery Source)</string>
   </property>
   <property name="toolTip">
    <string>E (Battery Source)</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/resistor.png</normaloff>:/circuit_icons/resources/resistor.png</iconset>
   </property>
   <property name="text">
    <string>R (Resistor)</string>
   </property>
   <property name="toolTip">
    <string>R (Resistor)</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/capacitor.png</normaloff>:/circuit_icons/resources/capacitor.png</iconset>
   </property>
   <property name="text">
    <string>C (Capacitor)</string>
   </property>
   <property name="toolTip">
    <string>C (Capacitor)</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/inductor.png</normaloff>:/circuit_icons/resources/inductor.png</iconset>
   </property>
   <property name="text">
    <string>L (Inductior)</string>
   </property>
   <property name="toolTip">
    <string>L (Inductior)</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/select.png</normaloff>:/tools_icons/resources/tools/select.png</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/pencil.png</normaloff>:/tools_icons/resources/tools/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/erase.png</normaloff>:/tools_icons/resources/tools/erase.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/tools_icons/resources/tools/graph.png</normaloff>:/tools_icons/resources/tools/graph.png</iconset>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionGnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/circuit_icons/resources/ground.png</normaloff>:/circuit_icons/resources/ground.png</iconset>
   </property>
   <property name="text">
    <string>Ground</string>
   </property>
   <property name="toolTip">
    <string>Ground</string>
   </property>
   <property name="shortcut">
    <string>Shift+G</string>
   </property>
  </action>
  <action name="actionField_Size">
   <property name="text">
    <string>Field Size</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FieldView</class>
   <extends>QGraphicsView</extends>
   <header>field/fieldview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
